zio-http/src/test/scala/zhttp/service/CORSSpec.scala

```diff
@@ -11,9 +12,9 @@ object CORSSpec extends HttpRunnableSpec(8089) {
   val env = EventLoopGroup.auto() ++ ChannelFactory.auto ++ ServerChannelFactory.auto
 
   val app: ZManaged[EventLoopGroup with ServerChannelFactory, Nothing, Unit] = serve {
-    CORS(HttpApp.collect { case Method.GET -> !! / "success" =>
+    HttpApp.collect { case Method.GET -> !! / "success" =>
       Response.ok
-    })
+    } @@ HttpMiddleware.cors()
```

Comment:
```suggestion
    } @@ cors
```
Some syntax improvements :)

Reply:
We can make it as `cors()` when `zhttp.http.middleware.HttpMiddleware.cors` is in imports. `cors` will not work because there is parameter in this function (it has default value, but still invocation needs `()`)